docs/ref/django-admin.txt

```diff
@@ -1513,6 +1513,13 @@ installed, ``ipdb`` is used instead.
 Discards output (``stdout`` and ``stderr``) for passing tests, in the same way
 as :option:`unittest's --buffer option<unittest.-b>`.
 
+.. django-admin-option:: --no-faulthandler
+
+.. versionadded:: 3.1
+
+Disables the :py:mod:`faulthandler` module, which will print tracebacks
+if a test aborts unexpectedly, during testing.
```

Comment:
I suggest rearranging to first document the feature, then explain the flag. Also we can point at `faulthandler.enable()` directly, to point out a bit that it's not using faulthandler timeouts.

```suggestion
Django automatically calls :func:`faulthandler.enable()` at the start of tests,
which allows it to print a traceback if the interpreter crashes. Pass
``--no-faulthandler`` to disable this behaviour.
```